contracts/gateway/EIP20Gateway.sol

```diff
@@ -1028,6 +1031,20 @@ contract EIP20Gateway is GatewayBase {
         success_ = true;
     }
 
+    /**
+     * @notice Returns the value of valueToken.
+     *
+     * @dev This function supports previous versions of this contract's ABI
+     *      that expect a public function, token, that returns the address
+     *      of the value token.
+     */
+    function token()
+        external
+        view
+        returns (address valueToken_)
+    {
+        return address(valueToken);
```

Comment:
Change one of two:

Line 1044:
```solidity
return (address)
```

Line 1046:
```solidity
valueToken_ = address(valueToken);
```

Reply:
We follow the 2nd approach in `EIP20Token`, so I will go with that.